pkg/bootstrap/docker/dockerhelper/helper.go

```diff
@@ -257,3 +257,18 @@ func (h *Helper) ListContainerNames() ([]string, error) {
 	}
 	return names, nil
 }
+
+// UserNamespaceEnabled checks whether docker daemon is running in user
+// namespace mode.
+func (h *Helper) UserNamespaceEnabled() (bool, error) {
+	info, err := h.dockerInfo()
+	if err != nil {
+		return false, err
+	}
+	for _, val := range info.SecurityOptions {
+		if val == "name=userns" {
```

Comment:
This check depends on the format of `SecurityOptions` and could be broken by updates in Docker but I think that I couldn't do anything better than this. Using `strings.Contains()` won't make this check better IMHO.